bHY6yh-de1103/cod-pu6y7d_bHY6yh-de1103.xml

```diff
@@ -60,43 +60,43 @@
     </p>
             <p xml:id="bHY6yh-de1103-d1e119">
                <quote xml:id="bHY6yh-de1103-Qd1e121">
-                  <lb ed="#V" n="37"/>Iob hystoria 34. c. Opus ho
+                 <ref> <lb ed="#V" n="37"/>Iob hystoria 34. c. </ref> Opus ho
```

Comment:
This looks close, but notice you've put the <ref> inside the quotation. 
And the <ref> should come after the line break (since the ref starts on this line, not before it.

This should look as follows:
```
                <lb ed="#V" n="37"/> <ref>Iob hystoria 34. c. </ref> <quote xml:id="bHY6yh-de1103-Qd1e121">Opus...